src/components/ETaxInvoiceForm.vue

```diff
@@ -43,6 +35,9 @@
         }
       };
     },
+    components: {
+      sellerinfoform: SellerInfoForm
```

Comment:
จริง ๆ เราใช้เป็น
components: {
  SellerInfoForm
}
ได้เลยนะตอนเรียกใช้ใน template ก็เรียกเป็น seller-info-form
ตามนี้เลย https://vuejs.org/v2/style-guide/#Component-name-casing-in-templates-strongly-recommended